### PR TITLE
[CodingStyle] Register InlineSimplePropertyAnnotationRector to coding-style config set

### PR DESCRIPTION
**Pull request by samsonasik**
Register InlineSimplePropertyAnnotationRector added by @dorrogeray at PR https://github.com/rectorphp/rector-src/pull/2070 to coding-style config set.

### CODE REVIEW

File: packages/NodeNameResolver/Regex/RegexPatternDetector.php [hunk 1]
```diff
@@ -10,8 +10,6 @@ final class RegexPatternDetector
 {
     /**
      * @var string[]
-     *
-     * This prevents miss matching like "aMethoda"
```
Comment (samsonasik) [inline packages/NodeNameResolver/Regex/RegexPatternDetector.php]: @dorrogeray the comments is somehow removed, it should be kept.

File: packages/NodeTypeResolver/Node/AttributeKey.php [hunk 2]
```diff
@@ -114,7 +114,6 @@ final class AttributeKey
 
     /**
      * @var string
-     * Use often in php-parser
```
Comment (samsonasik) [inline packages/NodeTypeResolver/Node/AttributeKey.php]: @dorrogeray the comments is somehow removed, it should be kept.

File: rules/Naming/Naming/PropertyNaming.php [hunk 3]
```diff
@@ -123,11 +123,7 @@ public function fqnToVariableName(ThisType | ObjectType | string $objectType): s
         }
 
         $className = $this->resolveClassName($objectType);
-        if (str_contains($className, '\\')) {
-            $shortClassName = (string) Strings::after($className, '\\', -1);
-        } else {
-            $shortClassName = $className;
-        }
+        str_contains($className, '\\') ? (string) Strings::after($className, '\\', -1) : $className;
```
Comment (samsonasik) [inline rules/Naming/Naming/PropertyNaming.php]: @dorrogeray the if else assign somehow changed to ternary without assign, it should be reverted.

File: packages/NodeNameResolver/Regex/RegexPatternDetector.php (hunk 1 above)
Comment (samsonasik) [reply packages/NodeNameResolver/Regex/RegexPatternDetector.php]: I create PR https://github.com/rectorphp/rector-src/pull/2073 for it.

File: rules/Naming/Naming/PropertyNaming.php (hunk 3 above)
Comment (samsonasik) [reply rules/Naming/Naming/PropertyNaming.php]: it seems happen when combined with `SimplifyIfElseToTernaryRector`

```diff
➜  rector-src git:(register-inline-simple-property-anno) ✗ bin/rector process rules/Naming/Naming/PropertyNaming.php --dry-run --clear-cache

 1/1 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%
1 file with changes
===================

1) rules/Naming/Naming/PropertyNaming.php:27

    ---------- begin diff ----------
@@ @@
  */
 final class PropertyNaming
 {
-    /**
-     * @var string[]
-     */
+    /** @var string[] */
     private const EXCLUDED_CLASSES = ['#Closure#', '#^Spl#', '#FileInfo#', '#^std#', '#Iterator#', '#SimpleXML#'];

-    /**
-     * @var string
-     */
+    /** @var string */
     private const INTERFACE = 'Interface';

     /**
@@ @@
         }

         $className = $this->resolveClassName($objectType);
-        if (str_contains($className, '\\')) {
-            $shortClassName = (string) Strings::after($className, '\\', -1);
-        } else {
-            $shortClassName = $className;
-        }
+        str_contains($className, '\\') ? (string) Strings::after($className, '\\', -1) : $className;

         $variableName = $this->removeInterfaceSuffixPrefix($shortClassName, 'interface');
         $variableName = $this->removeInterfaceSuffixPrefix($variableName, 'abstract');
    ----------- end diff -----------

Applied rules:
 * SimplifyIfElseToTernaryRector
 * VarConstantCommentRector
 * ConsistentPregDelimiterRector
 * InlineSimplePropertyAnnotationRector
 * SymplifyQuoteEscapeRector
 * RemoveUnusedVariableAssignRector
 * RegexDashEscapeRector```

Comment (dorrogeray) [reply rules/Naming/Naming/PropertyNaming.php]: Sorry, I am going on a holiday now so I will not be able to look at this until next week.

Comment (samsonasik) [reply rules/Naming/Naming/PropertyNaming.php]: ok, I will look into it.

Comment (samsonasik) [reply rules/Naming/Naming/PropertyNaming.php]: I created PR https://github.com/rectorphp/rector-src/pull/2074 for it.

**Comment by samsonasik** [comment]
rebased.

**Comment by samsonasik** [comment]
This is strange that on rectify, its diff is many files https://github.com/rectorphp/rector-src/runs/6028320279?check_suite_focus=true#step:6:27 but it only change 2 files in commit rectify  https://github.com/rectorphp/rector-src/pull/2072/commits/909ba6a80d503c4b5f945358a6a6e025b6d2054a

**Comment by samsonasik** [comment]
Ok, so it conflict with `fix-cs` command so it will always rollback in various paths:

```diff
 final class WorkerCommandLineFactoryTest extends AbstractKernelTestCase
 {
-    /** @var string */
+    /**
+     * @var string
+     */
     private const COMMAND = 'command';

-    /** @var string */
+    /**
+     * @var string
+     */
     private const DUMMY_MAIN_SCRIPT = 'main_script';
```

**Comment by samsonasik** [comment]
I guess it is ok to not register to `coding-style` config set, let's user to use it by register to their own rector.php config as it conflict change always rolled back by ecs.

I am closing it ;)

**Comment by dorrogeray** [comment]
> I guess it is ok to not register to `coding-style` config set, let's user to use it by register to their own rector.php config as it conflict change always rolled back by ecs.
> 
> I am closing it ;)

Agreed, this should be optional :+1:

**Pull request closed without merging**